ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -1301,6 +1301,35 @@ bool InputIsStatic(const Node* node, int index) {
   return std::find(inputs.begin(), inputs.end(), index) != inputs.end();
 }
 
+Status GetStaticInputs(Graph* graph, std::vector<int32>* static_input_indexes) {
```

Comment:
Minor. We generally use references for outputs. Then we wont have to check if the input pointer is null

Reply:
Ok, I see, its in keeping with the already existing `void GetStaticInputs(const Node* node, std::vector<int32>* inputs)`